_search-plugins/caching/tiered-cache.md

```diff
@@ -41,42 +41,44 @@ In OpenSearch 2.13, a request cache can use a tiered cache. To begin, configure
 
 ### Cache store name
 
-Set the cache store name to `tiered_spillover` to use the OpenSearch-provided tiered spillover cache implementation:
+To use the OpenSearch-provided tiered spillover cache implementation, set the cache store name to `tiered_spillover`:
 
 ```yaml
-indices.request.cache.store.name: tiered_spillover: true
+indices.request.cache.store.name: tiered_spillover
 ```
 {% include copy.html %}
 
 ### Setting on-heap and disk store tiers
 
-The `opensearch_onheap` setting is the built-in on-heap cache available in OpenSearch. The `ehcache_disk` setting is the disk cache implementation from [Ehcache](https://www.ehcache.org/). This requires installing the `cache-ehcache` plugin:
+Set the on-heap and disk store tiers as follows:
 
 ```yaml
 indices.request.cache.tiered_spillover.onheap.store.name: opensearch_onheap
 indices.request.cache.tiered_spillover.disk.store.name: ehcache_disk
 ```
 {% include copy.html %}
 
-For more information about installing non-bundled plugins, see [Additional plugins]({{site.url}}{{site.baseurl}}/install-and-configure/plugins/#additional-plugins).
+The `opensearch_onheap` setting is the built-in on-heap cache available in OpenSearch. 
```

Comment:
The setting can't literally be the cache.

Reply:
Reworded.